spec/ttml3.xml

```diff
@@ -1147,6 +1147,16 @@ attribute of a <loc href="#embedded-content-vocabulary-source"><el>source</el></
 content element</loc>.</p>
 </def>
 </gitem>
+<gitem id="terms-external-module">
+<label>[external module]</label>
```

Comment:
I would prefer to use a term that makes clear that the module is a specification, rather than some modular part of a document instance, for example. Also, in principle (according to the current draft charter anyway) the base whole specification consists of any base documentation and all of the modules, so this should be taken into account. The point here is that "external" carries connotations that may not be desirable.

Suggestion: `specification module`

Reply:
We cannot do as you suggest because we already have been using the term module within TTML since the beginning. Furthermore, I disagree that a module is a specification. It is a collection of functionality and the specification of that functionality. Depending upon the context of use, one may be emphasizing the former or the latter or both. If the draft charter uses this term in a misleading fashion, then we need to fix the draft charter, which I will take a closer look at in this regard.

Reply:
Sorry, looking back at this my words I accidentally put a confusing word "base" in, which I'll remove. The construct I think we need is that:

specification = base + modules

which I think we're in agreement about.

Reply:
I don't think so, since I don't know what "base" means. I think we often use "specification" to refer to a REC track document, to refer to other non-REC track W3C documents, to refer to non-W3C documents, and to refer to a combination of prose and/or formal language that appears in some larger context.

Reply:
As a REC-track document, TTML3 is a specification (with or without related) external modules. And each REC-track document that defines an external module is (will be) a specification. And other (non-W3C) documents that define external modules are (will be) specifications.

Reply:
@skynavga informally speaking, I agree. Formally, I'm proposing, in a similar way to CSS, that "the specification" is one deliverable, formed from a set of documents. Each of those documents is a technical report on the Recommendation track, in W3C Process terms.

Looking back at this thread (especially my comment that began it) I think what I wrote is confusing, and seems to be self-contradictory, so apologies for that.

Having re-read the existing spec text about modules, which had not been forefront in my mind on first review, I'm going to mark this thread as resolved since I can accept the text as is.